config/src/main/java/org/opennms/integration/api/xml/schema/poller/ServiceXml.java

```diff
@@ -0,0 +1,250 @@
+/*******************************************************************************
+ * This file is part of OpenNMS(R).
+ *
+ * Copyright (C) 2022 The OpenNMS Group, Inc.
+ * OpenNMS(R) is Copyright (C) 1999-2022 The OpenNMS Group, Inc.
+ *
+ * OpenNMS(R) is a registered trademark of The OpenNMS Group, Inc.
+ *
+ * OpenNMS(R) is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License,
+ * or (at your option) any later version.
+ *
+ * OpenNMS(R) is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with OpenNMS(R).  If not, see:
+ *      http://www.gnu.org/licenses/
+ *
+ * For more information contact:
+ *     OpenNMS(R) Licensing <license@opennms.org>
+ *     http://www.opennms.org/
+ *     http://www.opennms.com/
+ *******************************************************************************/
+
+package org.opennms.integration.api.xml.schema.poller;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlAttribute;
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlRootElement;
+
+/**
+ * Service to be polled for addresses in this
+ *  package.
+ */
+
+@XmlRootElement(name="service")
+@XmlAccessorType(XmlAccessType.NONE)
+public class ServiceXml implements Serializable {
+
+    /**
+     * Service name
+     */
+    @XmlAttribute(name="name")
+    private String m_name;
+
+    /**
+     * Interval at which the service is to be polled
+     */
+    @XmlAttribute(name="interval")
+    private Long m_interval;
+
+    /**
+     * Specifies if the service is user defined. Used specifically for UI
+     * purposes.
+     */
+    @XmlAttribute(name="user-defined")
+    private String m_userDefined = "false";
+
+    /**
+     * Status of the service. The service is polled only if this is set to
+     * 'on'.
+     */
+    @XmlAttribute(name="status")
+    private String m_status = "on";
```

Comment:
These are unused and should be removed.

Reply:
Which one of that 2 should be deleted? Both?
userDefined also exists in org.opennms.integration.api.v1.config.poller.Service. should it be also deleted from it?
status = "on" should be mapped to org.opennms.integration.api.v1.config.poller.Service.isEnabled() (I haven't added it jet). Sould isEnabled() also be removed?